### PR TITLE
RBAC: Update TestIntegrationFolderService remove RBAC setting

### PR DESCRIPTION
**Pull request by eleijonmarck**
**What is this feature?**
- Removes the RBAC setting from tests in the folder_service_test.go
- Unclear about this test, removed the test that was testing Accesscontrol settings in the folder_service, was added in this PR https://github.com/grafana/grafana/pull/46380/files#diff-a2a2f2de680c9563c671187770ff5397df8fd982bae7faefc343e7db63dd2217R40

Usage: "Fixes #<issue number>", or "Fixes (paste link of issue)"
https://github.com/grafana/grafana-enterprise/issues/4976

### CODE REVIEW

File: pkg/services/folder/folderimpl/folder_test.go [hunk 1]
```diff
@@ -37,19 +36,6 @@ import (
 var orgID = int64(1)
 var usr = &user.SignedInUser{UserID: 1, OrgID: orgID}
 
-func TestIntegrationProvideFolderService(t *testing.T) {
```
Comment (eleijonmarck) [inline pkg/services/folder/folderimpl/folder_test.go:40]: removed this test as it is testing Accesscontrol registering Scoperesolvers

Comment (eleijonmarck) [reply pkg/services/folder/folderimpl/folder_test.go:40]: i understand that it is part of the folder service, on how many scoped resolvers it should get

Comment (eleijonmarck) [reply pkg/services/folder/folderimpl/folder_test.go:40]: can add it back if we do not want to remove it

Comment (IevaVasiljeva) [reply pkg/services/folder/folderimpl/folder_test.go:40]: I think we should leave this test in. It checks if the RBAC scope resolvers are correctly registered, so it doesn't hurt to have it.

**Review by IevaVasiljeva — APPROVED** [review]
Awesome, if the tests pass with minimum changes we're all good! I think it's probably because we mock the guardian, which is where all the access control checks are done for both legacy access control and RBAC.

I think it's good to merge after putting that one test back in 👍

**Pull request merged**